cli/command/environment.go

```diff
@@ -13,5 +18,264 @@ func NewEnvironmentCommand(b *CommandBase) *EnvironmentCommand {
 }
 
 func (e *EnvironmentCommand) Command() cli.Command {
-	return cli.Command{}
+	return cli.Command{
+		Name:  "environment",
+		Usage: "manage layer0 environments",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Usage:     "create a new environment",
+				Action:    e.create,
+				ArgsUsage: "NAME",
```

Comment:
[nit] We were having a discussion about the ArgsUsage `NAME` here becoming `ENVIRONMENT_NAME`. I think this is the currently agreed consensus on the discussions I saw about this.